openassessment/xblock/static/js/fixtures/templates.json

```diff
@@ -488,6 +488,8 @@
                     "points_possible": 2
                 }
             ],
+            "feedbackprompt": "Feedback default prompt",
```

Comment:
I see this as `feedback_prompt` and `feedbackPrompt` elsewhere in the PR. why is it `feedbackprompt` here?

Reply:
There appears to be a naming inconsistency between the way data is stored (Rubric JSON and XML) and the variable in context used to populate the template, which uses `feedbackprompt`: https://github.com/edx/edx-ora2/blob/a82c2cdc6804d36642279959fe2d27a94882fc1b/openassessment/templates/openassessmentblock/edit/oa_edit_rubric.html#L38

Reply:
wonderful

Reply:
js expects camelCase while python expects snake_case, which results in this kind of mismatch rather often unfortunately.